package.json

```diff
@@ -19,6 +19,11 @@
   "bugs": {
     "url": "https://github.com/AssemblyScript/assemblyscript/issues"
   },
+  "engines": {
+    "node": ">=16.0.0",
+    "npm": ">=7.0.0"
+  },
+  "engineStrict": true,
```

Comment:
With these settings, what exactly will happen if someone on, say Node 15, installs the module? Will they get an outdated version that didn't specify the engines field? Or will there just be a warning? Or even an error?

Reply:
I see. So there is no mechanism in npm that would pick up an older version of the package when using an older node version? I believe that I have seen something like that in the past, but it's possible that my memory is off.

Reply:
As I remember we did it for `asinit` and loader esm / non-esm. But if we are going to use binaryen with bulk memory we need at least node v16 in any case

Reply:
What I am worried about is a case where a user is still on Node 15, and will forever get AS 0.20 or so, without noticing that there is already a more recent version, and we'll then see always the same questions. Might well be that this is not how it works, but I'm not sure.

Reply:
In this case I think it's fine to add :)